data/test/scenarios/interrupts.cfg

```diff
@@ -19,6 +27,15 @@
     [/event]
 )}
 
+#####
+# API(s) being tested: [return]
+##
+# Actions:
+# Break out of an infinite loop with [return]
+##
+# Expected end state:
+# The loop exits.
```

Comment:
Expectation is that the loop exits and the rest of the `[event]` is skipped. Note the `{FAIL}` after the `[/while]`.

Reply:
Updated.